lib/availability_resolver.js

```diff
@@ -162,6 +146,12 @@ class AvailabilityResolver {
             item.requestable = [DeliveryLocationsResolver.eddRequestableByOnSiteCriteria(item)]
           }
         }
+        // If requestable has not been already set during this serialization,
+        // set it based on separate requestable truthiness
+        if (!Feature.enabled('on-site-edd', request)) {
```

Comment:
Why check this feature? The 'on-site-edd' feature should generally be enabled in our deployments, and if it's disabled, that should only effect the `eddRequestable` for on-site (non-ReCAP) items.

(To enable this feature locally, set `FEATURES=on-site-edd[,...]` in your config)